plugin.video.drnu/resources/lib/addon.py

```diff
@@ -61,6 +61,15 @@ def __init__(self, plugin_url, plugin_handle):
         self.cache_path = xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('profile'))
         if not os.path.exists(self.cache_path):
             os.makedirs(self.cache_path)
+        buggalo.EMAIL_CONFIG = {
```

Comment:
I checked with the team, can you please make this opt-in instead of enabled by default? Emailing the author without the user consent is not a great idea. Or is the user prompted to do it internally in buggalo?

Reply:
Yes they are prompted in buggalo and the mail is a mail made for the addon so hopefully future developers can take it over and also look back in old bug reports if needed